### ISSUE TITLE
Ajout d'un nouvel Endpoint /agences/iot

### ISSUE THREAD

**Issue by Supervinh** [adriens/temps-attente-matrix-led#40]
# :grey_question: Contexte

L'Endpoint /agences renvoie une trop grande quantité de données pour la RAM de la Raspberry. 

# 🎯 Action

Il a été rajouté un nouvel endpoint /agences/iot qui ne renvoie qu'un minimum de données afin de ne pas surcharger la RAM.

```json
[
  {
    "idAgence": 1151,
    "designation": "Centre de Traitement Postal"
  },
  {
    "idAgence": 1152,
    "designation": "Agence de NOUMEA VALLEE DU TIR"
  }
]
```

**Comment by adriens** [comment]
Cadeau de Noël @raphb3383 @josegoue993

**Comment by raphb3383** [comment]
Bonjour @adriens 

Merci pour les infos, je vais reprendre le script afin d'utiliser ce nouvel EndPoint.

**Comment by adriens** [comment]
:crossed_fingers: :rocket:

**Comment by raphb3383** [comment]
Bonsoir @adriens,

Le nouvel EndPoint est bien utilisé dans le script en lieu et place d'un fichier .env pour lister les agences.

**Comment by adriens** [comment]
yes, bien joué, je confirme je l'ai également utilisée, yaura bientot un query parameter qui permettra de récupérer uniquement les agences avec de la data
